fenix/app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -261,143 +261,110 @@ class SettingsFragment : PreferenceFragmentCompat() {
+                SettingsFragmentDirections.actionSettingsFragmentToSitePermissionsFragment()
+            }
+
             resources.getString(R.string.pref_key_delete_browsing_data) -> {
                 SettingsFragmentDirections.actionSettingsFragmentToDeleteBrowsingDataFragment()
             }
+
             resources.getString(R.string.pref_key_delete_browsing_data_on_quit_preference) -> {
                 SettingsFragmentDirections.actionSettingsFragmentToDeleteBrowsingDataOnQuitFragment()
             }
+
             resources.getString(R.string.pref_key_notifications) -> {
                 context?.navigateToNotificationsSettings {}
                 null
             }
-            resources.getString(R.string.pref_key_customize) -> {
-                SettingsFragmentDirections.actionSettingsFragmentToCustomizationFragment()
-            }
-            resources.getString(R.string.pref_key_privacy_link) -> {
-                val intent = SupportUtils.createCustomTabIntent(
-                    requireContext(),
-                    SupportUtils.getMozillaPageUrl(SupportUtils.MozillaPage.PRIVATE_NOTICE),
-                )
-                startActivity(intent)
-                null
-            }
-            resources.getString(R.string.pref_key_your_rights) -> {
-                val context = requireContext()
-                val intent = SupportUtils.createCustomTabIntent(
-                    context,
-                    SupportUtils.getSumoURLForTopic(context, SupportUtils.SumoTopic.YOUR_RIGHTS),
-                )
-                startActivity(intent)
-                null
-            }
-            resources.getString(R.string.pref_key_debug_settings) -> {
-                SettingsFragmentDirections.actionSettingsFragmentToSecretSettingsFragment()
-            }
-            resources.getString(R.string.pref_key_secret_debug_info) -> {
-                SettingsFragmentDirections.actionSettingsFragmentToSecretInfoSettingsFragment()
+
+            resources.getString(R.string.pref_key_data_choices) -> {
+                SettingsFragmentDirections.actionSettingsFragmentToDataChoicesFragment()
             }
-            resources.getString(R.string.pref_key_nimbus_experiments) -> {
-                SettingsFragmentDirections.actionSettingsFragmentToNimbusExperimentsFragment()
+
+            /* Advanced preferences */
+            resources.getString(R.string.pref_key_addons) -> {
+                Addons.openAddonsInSettings.record(mozilla.components.service.glean.private.NoExtras())
+                SettingsFragmentDirections.actionSettingsFragmentToAddonsFragment()
             }
+            // Only displayed when secret settings are enabled
             resources.getString(R.string.pref_key_install_local_addon) -> {
                 addonFilePicker.launch()
                 null
             }
+            // Only displayed when secret settings are enabled
```

Comment:
Could we also add a new line in-between for these since we did it for everything else? This also needs to happen below so you may need to go through your changes again.

Reply:
Good spot, the formatter must've ignored because of the comments